plugins/tiledb/io/TileDBReader.cpp

```diff
@@ -34,8 +34,17 @@
 
 #include <algorithm>
 
+#include <nlohmann/json.hpp>
+
 #include "TileDBReader.hpp"
 
+
+const char pathSeparator =
```

Comment:
FIXME: When we go to C++17, we should use std::filesystem:: preferred_separator

Reply:
:+1: